lib/Tool/Console.php

```diff
@@ -364,7 +364,7 @@ protected static function execInBackgroundWindows($cmd, $outputFile)
      *
      * @return array|string
      */
-    public static function addLowProcessPriority($cmd)
+    public static function addLowProcessPriority(&$cmd)
```

Comment:
maybe we get rid of return statement since it is not used anywhere and it is not a BC break as the method is marked as internal...could you please change it? thanks!

Reply:
return value is removed